src/libs/ReportUtils.ts

```diff
@@ -2641,19 +2641,24 @@ function isOneOnOneChat(report: OnyxEntry<Report>): boolean {
 
 function isPayer(session: OnyxEntry<Session>, iouReport: OnyxEntry<Report>, onlyShowPayElsewhere = false, reportPolicy?: OnyxInputOrEntry<Policy>) {
     const policy = reportPolicy ?? allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${iouReport?.policyID}`] ?? null;
-    const policyType = policy?.type;
-    const isAdmin = policyType !== CONST.POLICY.TYPE.PERSONAL && policy?.role === CONST.POLICY.ROLE.ADMIN;
+
+    if (!policy) {
+        return false;
```

Comment:
I think we need to log a warn here https://github.com/Expensify/App/blob/469d0a48daff713bf8f44a77a653896c7ab77cbb/src/libs/ReportUtils.ts#L4908 it is a bad practice to just return silently, without letting the user or us know that this happened